docs/user-guide/host-settings.md

```diff
@@ -0,0 +1,194 @@
+# Host Settings
+This page contains settings for Linux servers, it agnostic of the Linux
+distribution. The commands use the `ip` utility from the `iproute2` package.
+These settings do not persist across reboots. Consult the documentation from
+the Linux distribution for guides on how to persist the settings in for example
+[Netplan](https://documentation.ubuntu.com/server/explanation/networking/configuring-networks/index.html) or 
+[Network Manager](https://networkmanager.dev/docs/admins/). For additional
+details consult the [kernel bonding
+driver](https://www.kernel.org/doc/html/latest/networking/bonding.html)
+documentation.
+
+## MCLAG / ESLAG
+
+The multi-chassis LAG architecture is a way to provide device redundancy
+in a network architecture. At the physical layer an MCLAG topology is a single
+server connected to two different switches and those switches are directly connected
+to each other in addition to being connected to the rest of the fabric.
+
+ESLAG is a similar topology as MCLAG with the beneficial difference that the
+switches do not need a to be connected to each other. There can be up to 4
+switches in an ESLAG group, whereas MCLAG is always 2 switches.
+
+The standard that governs this technology is commonly referred to as 802.3ad.
+
+### Server Settings
+
+The supported server side setting for bonding is `802.3ad`, with hashing policy
+`layer2` or `layer2+3`. To configure this in Linux:
+``` bash
+sudo ip link add $bond_name type bond miimon 100 mode 802.3ad layer2
+sudo ip link set $slave1_name master $bond_name
+sudo ip link set $slave2_name master $bond_name
+```
+
+The `miimon 100` indicates that the  bonding driver will use miimon to
+determine if the link is `up` and it will poll every 100 milliseconds.
+
+`layer2` or `layer2+3` is the hashing mode, which controls the input into the
+hash function. See the [kernel bonding
+driver](https://www.kernel.org/doc/html/latest/networking/bonding.html)
+documentation for additional details.
+
+After the bond has been created, it can receive a dhcp address, or be
+statically assigned:
+
+`sudo ip addr add 10.10.10.3/24 dev bond0`
```

Comment:
Again, how did you pick these IPs, can we somehow make more clear that they are example IPs?